linera-chain/src/chain.rs

```diff
@@ -537,6 +537,13 @@ where
         }
     }
 
+    pub async fn next_height_to_preprocess(&self) -> Result<BlockHeight, ChainError> {
```

Comment:
Can you add a comment with the explanation why do it this way? I've found myself wondering what many of our pub methods do.